PPP_Robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
#pid: {p: 100.0, i: 0.01, d: 10.0}
  # Position Controllers ---------------------------------------
  x_prismatic_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: x_prismatic_joint
    pid: {p: 5000.0, i: 0.0, d: 100.0}
  z_prismatic_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: z_prismatic_joint
    pid: {p: 5000.0, i: 0.0, d: 100.0}
  y_prismatic_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: y_prismatic_joint
    pid: {p: 5000.0, i: 0.0, d: 100.0}